recipes-extended/iptables/iptables/ip6tables.rules

```diff
@@ -1,3 +1,9 @@
 *filter
+-A INPUT -i lo -p tcp -m tcp -j ACCEPT
+-A INPUT -i lo -p udp -m udp -j ACCEPT
+-A INPUT -m conntrack --ctstate ESTABLISHED,RELATED -j ACCEPT
 -A INPUT ! -i enp+ -p tcp -m tcp --dport 22 -j DROP
+-A INPUT ! -i enp+ -p tcp -m tcp --dport 5000 -j DROP
+-A INPUT ! -i enp+ -p tcp -m tcp --dport 6653 -j DROP
+-A INPUT ! -i enp+ -p tcp -m tcp --dport 50051 -j DROP
```

Comment:
As the file grows in size, it might be nice to have some comments here, especially since port 22 might be common knowledge to be ssh, the other ones aren't.